### ISSUE TITLE
Javascript error in main on playback of song

### ISSUE THREAD

**Issue by manaaan** [jeffvli/sonixd#421]
**Describe the bug**
Whenever a song starts or finishes, I get an error message as linked below in the screenshot.
When skipping to the next song, the error appears only once. If letting the song naturally take its course and expecting to start the next song, it appears twice.
Sometimes it stops the playback on the error, but mostly it still continues to play the next song.

**To Reproduce**
1. Go to 'Albums'
2. Select an Album (Happens not exclusively when playing from 'Albums', but from any view that allows to play a song)
3. Play a song 
4. See error (but song starts playing)

**Expected behavior**
Expectation would be no error, but just the playback of the song.

**Screenshots**
```
A JavaScript error occurred in the main process

Uncaught Exception:
TypeError: r.id?.replace is not a function
  at IpcMainImpl.<anonymous> (/tmp/-appimage_extracted_5e9afda397eee12d433d9ff2cba105c1/resources/-app.asar/main.prod.js:2:1145547)
  at IpcMainImpl.emt (events.js:315:20)
  at Object.<anonymous> (electron/js2c/browser_init.js:161:9492)
  at Object.emit (events.js:315:20)
```

**Desktop (please complete the following information):**
 - OS: Linux Mint 20.3
 - Version 0.15.3

**Additional context**
Running from `Sonixd-0.15.3-linux-x86_64.AppImage`.
Fetching songs from a funkwhale server through their subsonic API.

It seems there is an `ID` to be set per default, which is however not present in my case. Not sure how I can add this `ID` retrospectively to my music data.
I have a similar error when trying to save changes to a playlist, where I get an error message that an `ID` must be set on the playlist to save.

**Comment by jeffvli** [comment]
Pushed a fix in 0.15.4.

**Comment by Weissnix4711** [comment]
Seems to still be happening on 0.15.5

![image](https://github.com/jeffvli/sonixd/assets/42943391/8d5e81c4-9285-4031-8fb4-57777ddc3a08)
